.travis.yml

```diff
@@ -7,14 +7,16 @@ android:
     - platform-tools
     - tools
     - build-tools-24.0.2
+    - android-21
     - android-24
```

Comment:
When we try stick to only one version 24, there were problems on build.
In this case emulator image required 21. 
Our build target is 24 that's why we should add and 24